packages/ember/src/components/form/phone-field.gts

```diff
@@ -0,0 +1,120 @@
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { runTask } from 'ember-lifeline';
+
+import TextField from './text-field.gts';
+import onInsert from '../../modifiers/did-insert.ts';
+
+import type { Optional } from '../../types.d.ts';
+
+export default class PhoneField extends TextField {
+  @tracked
+  inputElement: Optional<HTMLInputElement> = null;
```

Comment:
If you're expecting this value to be updated within the first run loop, you can tell TypeScript that it'll be provided, even if it's outside of expected instantiation timing:
```suggestion
  inputElement!: HTMLInputElement;
```
Now you won't have to do any null checks in the rest of the component, unless it's in a place that you think might run before the `onInsert` modifier runs.